docs/reference/esql/esql-process-data-with-dissect-grok.asciidoc

```diff
@@ -248,6 +248,16 @@ include::{esql-specs}/docs.csv-spec[tag=grokWithEscape]
 ----
 ====
 
+For this reason, in general it is more convenient to use triple quotes `"""` for GROK patterns,
+that do not require escaping for backslash.
+
+[source.merge.styled,esql]
+----
+include::{esql-specs}/docs.csv-spec[tag=grokWithEscapeTripleQuotes]
+----
+====
```

Comment:
```suggestion
```

@luigidellaquila I think this might be the culprit 🤞 for docs CI fail

Reply:
Thanks @leemthompo!